internal/multigitter/run.go

```diff
@@ -323,8 +326,26 @@ func (r *Runner) runSingleRepo(ctx context.Context, repo scm.Repository) (scm.Pu
 
 	var pr scm.PullRequest
 	if existingPullRequest != nil {
-		log.Info("Skip creating pull requests since one is already open")
 		pr = existingPullRequest
+		if r.ConflictStrategy == ConflictStrategyReplace {
+			log.Info("Updating pull request since one is already open")
+			pr, err = r.VersionController.UpdatePullRequest(ctx, repo, pr, scm.NewPullRequest{
+				Title:         r.PullRequestTitle,
+				Body:          r.PullRequestBody,
+				Head:          r.FeatureBranch,
+				Base:          baseBranch,
+				Reviewers:     getReviewers(r.Reviewers, r.MaxReviewers),
+				TeamReviewers: getReviewers(r.TeamReviewers, r.MaxTeamReviewers),
```

Comment:
Thought/Discussion: If `max-reviewers` is less than the reviewers provided, the reviewers used will be randomized. Do we want this to happen in the case of an update or not? 🤔 I have no real opinion here, but a conscious decision has to be made.